include/srdfdom/model.h

```diff
@@ -201,15 +203,17 @@ class Model
     return name_;
   }
 
-  /// Get the list of pairs of links that need not be checked for collisions (because they can never touch given the
-  /// geometry and kinematics of the robot)
-  const std::vector<DisabledCollision>& getDisabledCollisionPairs() const
+  /// Get the list of links that should have collision checking disabled by default (and only selectively enabled)
+  const std::vector<std::string>& getNoCollisionLinks() const
```

Comment:
```suggestion
  const std::vector<std::string>& getNoDefaultCollisionLinks() const
```
... and of course the associated refactoring.